mlos_bench/mlos_bench/tunables/tunable_groups.py

```diff
@@ -108,18 +128,29 @@ def __repr__(self) -> str:
             for group in sorted(self._tunable_groups.values(), key=lambda g: (-g.cost, g.name))
             for tunable in sorted(group._tunables.values())) + " }"
 
-    def __getitem__(self, name: str) -> TunableValue:
+    def __contains__(self, tunable: Union[str, Tunable]) -> bool:
+        """
+        Checks if the given name/tunable is in this tunable group.
+        """
+        name: str = tunable.name if isinstance(tunable, Tunable) else tunable
+        return name in self._index
+
+    def __getitem__(self, tunable: Union[str, Tunable]) -> TunableValue:
```

Comment:
As with elsewhere, adds support for indexing by `Tunable` as well as `str` as a convenience.